inter_company_rules/README.rst

```diff
@@ -0,0 +1,68 @@
+
+Configuration
+=============
+
+To configure this module, you need to go to the menu *Settings > Companies > Companies*, select one of the companies and go to the tab *Inter-Company Rules*. You have to choose which scenario you want to have for interactions between companies: either automate the creation of sale.order / purchase.order between companies, or automate the creation of customer invoice / supplier invoice between companies (you cannot do both).
+
+Another important configuration is the *Inter Company User* : it is the user that will be used to automatically generate the corresponding object in the other company. One important thing to understand is that the fields *Customer Taxes* (technical field field: *taxes_id*) and *Supplier Taxes* (technical field field: *supplier_taxes_id*) on product.template are **NOT** property fields (in new API, we would say: *company_dependant=False*). Therefore, it is only the multi-company record rule on account.tax that make sure that, for a particular product, the taxes for company A are selected when you invoice this product in company A and the taxes for company B are selected when you invoice this product in company B. So you cannot select the administrator as *Inter Company User* because this user by-passes the record rules ; you have to select a regular user that is attached to the company and only to this company (don't select a user that is allowed to switch between companies). Special per-company filtering of taxes has been added in the *product_id_change* method in of sale.order.line and purchase.order.line (cf `this commit <https://github.com/odoo/odoo/commit/eb993b7f3bb93a951a63c5db73ca069ba8f835c3>`_ for the purchase.order.line, dated December 11th 2015), but this filtering is not yet present in the *product_id_change* method of account.invoice.line. When this per-company filtering of taxes will be present on account.invoice.line, we will be able to use *admin* as the *Inter Company User*.
+
+Usage
+=====
+
+If you choose the option *Create Invoices/Refunds when encoding invoices/refunds made to this company* in the configuration of company B, when you validate a *Customer Invoice* in company A with company B as customer, then a *Supplier Invoice* will be automatically created in company B with company A as supplier.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/133/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/multi-company/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+multi-company/issues/new?body=module:%20
+inter_company_rules%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```

Comment:
Can you follow this template?
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst